rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php

```diff
@@ -176,7 +176,7 @@ private function shouldSkip(string $classLikeName): bool
         }
 
         foreach ($this->classesToSkip as $classToSkip) {
-            if ($this->nodeNameResolver->isStringName($classLikeName, $classToSkip)) {
+            if ($this->nodeNameResolver->matchesStringName($classLikeName, $classToSkip)) {
```

Comment:
this requires a wildcard in the pattern now. should we call regular isStringName() beforehand to keep BC?

Reply:
Yeah, this should allow both types